afk/afk.go.tmpl

```diff
@@ -11,16 +11,15 @@
 */}}
 
 {{/* CONFIGURATION VALUE STARTS HERE */}}
-{{ $removeAfkOnMessage := true }}{{/* Remove the AFK database from an user if he/she types something in chat. */}}
+{{ $removeAfkOnMessage := false }}{{/* Remove the AFK database from an user if he/she types something in chat. */}}
```

Comment:
I would let this be the default setting:
```suggestion
{{ $removeAfkOnMessage := true}}{{/* Remove the AFK database from an user if he/she types something in chat. */}}
```

Reply:
Yes, reverting this change would be preferable. From feedback on the first version of AFK, many users were used to the AFK clearing when they sent a message.

Reply:
Agreed.

Reply:
Why was this resolved without any changes?